x-pack/plugins/licensing/server/license_feature.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { License } from './license';
+import { LicenseFeatureSerializer } from './types';
+
+export class LicenseFeature {
+  private serializable: LicenseFeatureSerializer = license => ({
+    name: this.name,
+    isAvailable: this.isAvailable,
+    isEnabled: this.isEnabled,
+  });
+
+  constructor(public name: string, private feature: any = {}, private license: License) {}
```

Comment:
- can we define `feature` interface? 
- `license` is not used in the file

Reply:
1. Fixed in upcoming PR.
2. It is used in serialization.